docs/config.rst

```diff
@@ -243,6 +243,23 @@ Features
 
    .. versionadded:: 2.0
 
+.. attribute:: LP_ENABLE_CONTAINER
+   :type: bool
+   :value: 0
+
+   Indicate if the shell is running in a container environment (e.g.
+   Docker, Podman, LXC, Singularity)
+
+   Note: Containers may inherit some or even no variables from their parent
+   shell, so this may behave inconsisently with different container software.
+   For example, Docker doesn't inherit anything unless explicitly told to.
+   Singularity in many configurations inherits may of variables and
+   functions, but removes the PROMPT_COMMAND that makes liquidprompt update.
```

Comment:
`PROMPT_COMMAND` is a feature of Bash. Zsh uses different hooks (specifically, hooks), while Ksh (not yet supported) uses the debug trap. I would drop this reference.

Rather more importantly than variables not making it across the container boundary is that functions never do. I don't see a way for Liquidprompt to make it inside a container without being specifically sourced in the new shell. Not that it kills this feature, but this note section should mention that (need to source LP inside the container) here.

Reply:
I have added an additional commit to this reflecting the requested documentation improvements.